media-api/app/lib/Config.scala

```diff
@@ -34,6 +34,7 @@ object Config extends CommonPlayAppConfig with CommonPlayAppProperties {
 
   val imageBucket: String = properties("s3.image.bucket")
   val thumbBucket: String = properties("s3.thumb.bucket")
+  val dataStoreBucket: String = properties("s3.datastore.bucket")
```

Comment:
OK. I think I'd have favoured co-opting a different less crucial service (maybe metadata-editor), in case it has any problematic side-effects.